src/components/Login.vue

```diff
@@ -13,26 +13,30 @@
             >
             <v-icon size="196px" class="pt-4" color="#e8e8e8"> mdi-account-circle </v-icon>
```

Comment:
Rather than have a static size, look into variable sizes for different device dimensions. (responsive images)